### PR TITLE
Update Run button code lens base on new response

### PR DESCRIPTION
**Pull request by sdankel**
This updates the client to handle the new response type for the `sway/runnables` LSP request type.

Related https://github.com/FuelLabs/sway/pull/2459/files

### CODE REVIEW

File: client/src/commands/forcBuild.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { window } from 'vscode';
+import { log } from '../util';
+
+export default function forcBuild(forcDir: string) {
+  exec(`cd ${forcDir} && forc run`, (error, stdout, _stderr) => {
+    if (error) {
+      window.showInformationMessage(`Failed: see output console for error`);
+      log.error(stdout);
+    } else {
+      g;
```
Comment (sdankel) [inline client/src/commands/forcBuild.ts:11]:
```suggestion
```

**Review by adlerjohn — COMMENTED** [review]
Hmmm. What's the rationale to put this only for predicates and not also scripts?

**Comment by sdankel** [comment]
> Hmmm. What's the rationale to put this only for predicates and not also scripts?

Scripts already have a "Run" button, as of https://github.com/FuelLabs/sway-vscode-plugin/pull/77

**Comment by sdankel** [comment]
Spoke with @JoshuaBatty offline - for now we're only going to support the run button for scripts. This PR is still necessary because of the new response type from the LSP.

**Review by JoshuaBatty — APPROVED** [review]

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**